addons/payment_stripe_sca/static/src/js/stripe_sca.js

```diff
@@ -0,0 +1,111 @@
+odoo.define('payment_stripe_sca.stripe_sca', function (require) {
+"use strict";
```

Comment:
since this is not widgetized, i'd rather avoid having this loaded for every page
can you add a DOM check for the JS module:
```js
if(!$('#pay_stripe').length) {
    return $.Deferred().reject("DOM doesn't contain '#pay_stripe'");
}
```
(or a better selector than `#pay_stripe`, I just suggest this based on the content of the file)